index.html

```diff
@@ -1,12 +1,261 @@
+                            <span><ion-icon class="categories-icon" name="phone-portrait-outline"></ion-icon></span>
+                            <span><ion-icon class="categories-icon" name="watch-outline"></ion-icon></span>
+                            <span><ion-icon class="categories-icon" name="desktop-outline"></ion-icon></span>
+                            <span><ion-icon class="categories-icon" name="bag-handle-outline"></ion-icon></span>
+                            <span><ion-icon class="categories-icon" name="barbell-outline"></ion-icon></span>
+                            <span><ion-icon class="categories-icon" name="basketball-outline"></ion-icon></span>
+                            <span><ion-icon class="categories-icon" name="bed-outline"></ion-icon></span>
+                            <span><ion-icon class="categories-icon" name="camera-outline"></ion-icon></span>
+                            <span><ion-icon class="categories-icon" name="bicycle-outline"></ion-icon></span>
+                            <span><ion-icon class="categories-icon" name="game-controller-outline"></ion-icon></span>
+                            <span><ion-icon class="categories-icon" name="phone-portrait-outline"></ion-icon></span>
+                            <span><ion-icon class="categories-icon" name="watch-outline"></ion-icon></span>
+                            <span><ion-icon class="categories-icon" name="desktop-outline"></ion-icon></span>
+                            <span><ion-icon class="categories-icon" name="bag-handle-outline"></ion-icon></span>
+                            <span><ion-icon class="categories-icon" name="barbell-outline"></ion-icon></span>
+                            <span><ion-icon class="categories-icon" name="basketball-outline"></ion-icon></span>
+                            <span><ion-icon class="categories-icon" name="bed-outline"></ion-icon></span>
+                            <span><ion-icon class="categories-icon" name="camera-outline"></ion-icon></span>
+                            <span><ion-icon class="categories-icon" name="bicycle-outline"></ion-icon></span>
+                            <span><ion-icon class="categories-icon" name="game-controller-outline"></ion-icon></span>
+                        </div>
+
+                        <h3 class="my-4">Hot Deals</h3>
+                        <div class="categories-section flex-row my-4">
+                            <div class="product mr-5">
+                                <img class="img-responsive" src="https://www.oberlo.com/media/1603954764-image059.png?w=1824&fit=max" alt="product">
+                                <div class="my-3">
+                                    <h4>Product Name</h4>
+                                    <p class="brand-name">Brand Name</p>
+                                    <div class="flex-align-center">
+                                        <span class="mr-3">
+                                            <label>
```

Comment:
Can you explain the use of `label` tag here?

Reply:
i am hiding the actual checkbox and showing star icon for rating. That's why used label to keep both of them inside it.

Reply:
I do not think that this is the correct use-case for `label` tag.

Reply:
ok this is working for me. I'll look into this Vishal thanks for reviewing